near-sdk/src/store/tree_map/iter.rs

```diff
@@ -294,6 +294,116 @@ where
     }
 }
 
+fn find_min<'a, K>(
+    tree: &'a Tree<K>,
+    root: Option<&FreeListIndex>,
```

Comment:
small nit: the root is coming from the tree, so there should be no need to pass in the root in this and `find_max`, right?

Reply:
I think its needed. Its the root of the `find_min` and not root of the tree. See below.

https://github.com/near/near-sdk-rs/blob/1351cad09653a8a333be7509fb9941fcbf79bc61/near-sdk/src/store/tree_map/iter.rs#L366

Reply:
oops! I missed that when I skimmed this